android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -184,12 +197,13 @@ public String getName() {
         }
     }
 
-    private void tryParseIsAudioMuted(Intent intent) {
+    private Boolean tryParseIsAudioMuted(Intent intent) {
```

Comment:
Why not return `boolean` and `false` in case of error?

Reply:
The thing is we need to differentiate between the actual false value in the intent and no value (null).

Reply:
Yes, but why do we care? If it's not specified we can treat is as false, which is the expected behavior, isn't it?

Reply:
When we explicitly get false, we modify the notification while on null I thing we do not want that. It might just work cause for the moment this will be null when starting the service or when getting the HANG_UP action (case in which it will not matter that we modify the notification cause it will be hidden soon), but I still think it's safer to just set the isMuted when we actually know we want to set it.

Reply:
Ok.